core/models/access_token.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils import timezone
+
+from django_cyverse_auth.models import Token
+
+
+class AccessToken(models.Model):
+    """
+    Extend the django_cyverse_auth Token to add a name
```

Comment:
This is literally describing what the model is, may explain how this is used in the context of the api.